pegasus/sites.v3/code.org/public/css/dance-landing/dance.css

```diff
@@ -2,6 +2,52 @@
   font-size: 20px;
 }
 
+.img-dance-stars {
+  position: absolute;
+  top: 10px;
+  left: 70px;
+  width: 330px;
+  padding: 20px 10px 10px;
+}
+
+.desktop-only, .tablet-only, .mobile-only {
+  display: none;
+}
+
+.starwars-tutorial.dance-tutorial {
+  top: 20px;
+}
+
+.dance-try-button {
+  font-size:18px;
+  height:40px;
+  transition: 300ms all;
+}
+
+.dance-try-button:hover {
+  background-color: #FFCC22;
+}
+
+.foot-note-text {
+  text-align: center;
+  position: absolute;
+  bottom: 0;
+  color: #00adbc;
+  text-align: center;
+  margin-bottom: 15px;
+  font: 10px/16px 'Gotham 5r', sans-serif;
```

Comment:
Interesting, what does this / do?

Reply:
font: 10px/16px 'Gotham 5r', sans-serif;

10px is the font size and 16px is the line height.  
The forward slash does not imply division, it is just a notation to separate the font size and the line height inside the font attribute.  However, I can see how it can be interpreted as a ratio where the first value if the font size and the second value is the vertical space it occupies.  The font size and line height work independently and each value does not modify the other value.

Here is a good description I found of this neat trick:
Shorthand property for setting 'font-style', 'font-variant', 'font-weight', 'font-size', 'line-height', and 'font-family', at the same place in the style sheet. The syntax of this property is based on a traditional typographical shorthand notation to set multiple properties related to fonts.

Reply:
@breville Thanks for the feedback:
Some quick notes:

The entire whitespace to the left of the translate link is also clickable. Only the link text itself should be.
The artist names look good but I wonder whether they're too small for accessibility reasons.
- Good point.  I adjusted the font size.

nit: the teacher resources & dance party unplugged boxes have a border, but when we do 
similar UI elsewhere (e.g. teacher announcement) we don't have a border. Not sure if this is universally true though.
- Removed the border from teacher resources' boxes.  Added the change to this [PR #25996](https://github.com/code-dot-org/code-dot-org/pull/25996)

The images don't fill the whole box for these when on narrow screens. It should be possible.
- Are you referring to the artists' heads?

"intend on studying" should be "intend to study"
- Updated i18n gsheet
Nice work handling lots of different types of elements with lots of different responsive 
behaviours.
- Thanks!